File: generator/output/ts-model/wallet.model.ts
class WalletModel {
  public id: number = 0;

  // deleted 
  public deleted: boolean = false;

  // user_id 
  public user_id: number = 0;

  // money 余额
  public money: number = 0;

  public created_at: string;

  public updated_at: string;
}

const WalletModelFields = {
  id: "id",
  deleted: "deleted",
  user_id: "user_id",
  money: "money",
  created_at: "created_at",
  updated_at: "updated_at"
};

export {WalletModel, WalletModelFields};

File: generator/output/entityModel/WalletModel.java
package com.licenseair.backend.domainModel;

import java.util.List;
import lombok.Data;

import java.lang.Long;
import java.math.BigDecimal;
import java.lang.Boolean;
import java.sql.Timestamp;

/**
* Created by licenseair.com
* Wallet
*/
@Data()
public class WalletModel {
  public Long id = null;

  /**
  * 
  */
  public Boolean deleted = null;

  /**
  * 
  */
  public Long user_id = null;

  /**
  * 余额
  */
  public BigDecimal money = null;

  public Timestamp created_at = null;

  public Timestamp updated_at = null;

}
